### DISCUSSION TITLE
Property 'sessionPath' of undefined

### DISCUSSION THREAD

**Discussion by themaxgross** [community/community#6131] [Codespaces] [answered]
Hello y'all!

I was enjoying CodeSpaces immensely until earlier today, when my VSCode decided to stop connecting to any CS instances and instead the extensin throws up an error: `Connect to Codespace: Cannot read property 'sessionPath' of undefined`.

I thought it might have been an extension issue, a cache issue, a network issue or a container issue, but restarting / resetting all of those didn't help. :\

Any ideas?

**Comment by matthewisabel** [comment, marked as answer]
This is part of a current incident. Fix for desktop connections is rolling out now, but should work from the web.

**Reply by themaxgross** [reply]
Oh! I thought it was a me thing. 😅. Thank you, and best wishes to y'all

**Reply by matthewisabel** [reply]
Should be fully resolved now

**Comment by sbs2001** [comment]
Facing the same issue suddenly.

**Reply by jkeech** [reply]
Apologies for this issue! This looks like a bug in a recent version of the extension when using the Codespaces: Connect to Codespace command in the command palette. A workaround is to connect via the Remote Explorer viewlet which won't hit the error. A fix for this issue will be included in the next version of the extension.

**Reply by jkeech** [reply]
GitHub Codespaces extension version 1.4.3 was just published with the fix. Please try updating the extension and let us know if you continue to see this issue. Thanks!
